resources/views/turnstileWidget.blade.php

```diff
@@ -0,0 +1,10 @@
+<div class="cf-turnstile"
+     data-sitekey="{{ config('services.turnstile.key') }}"
+     @if(isset($configurations['action'])) data-action="{{ $configurations['action'] }}" @endif
```

Comment:
Could this not all be done with a loop?

Reply:
Thanks for the feedback! I have made the changes. 😄